robot-server/robot_server/sessions/control_commands.py

```diff
@@ -0,0 +1,36 @@
+"""Request and response models for session input resources."""
+from datetime import datetime
+from enum import Enum
+from pydantic import BaseModel, Field
+
+from robot_server.service.json_api import ResourceModel
+
+
+class SessionControlType(str, Enum):
+    """Control input types."""
```

Comment:
```suggestion
    """Types of control commands."""
```

(As opposed to "control command types," where "control" could be misread as a verb instead of a noun.)